modules/configuring-ovnk-additional-networks.adoc

```diff
@@ -8,6 +8,11 @@
 
 The {openshift-networking} OVN-Kubernetes network plugin allows the configuration of secondary network interfaces for pods. To configure secondary network interfaces, you must define the configurations in the `NetworkAttachmentDefinition` custom resource (CR).
 
+[NOTE]
+====
+Pod and multi-network policy creation may remain in a pending state until the associated `network-attachment-definition` CR is processed by the OVN-Kubernetes control plane agent in the nodes.
```

Comment:
Could swap around to avoid passive:
```suggestion
Pod and multi-network policy creation may remain in a pending state until the OVN-Kubernetes control plane agent in the nodes processes the associated `network-attachment-definition` CR.
```